arangodb-net-standard.Test/CollectionApi/CollectionApiClientTest.cs

```diff
@@ -232,5 +232,27 @@ public async Task GetCollectionsAsync_ShouldSucceed()
             Assert.Equal(HttpStatusCode.OK, response.Code);
             Assert.NotNull(collectionExists);
         }
+
+        [Fact]
+        public async Task GetCollectionRevisionAsync_ShouldSucceed()
+        {
+            var response = await _collectionApi.GetCollectionRevisionAsync(_testCollection);
+            Assert.Equal(HttpStatusCode.OK, response.Code);
+            Assert.Equal(_testCollection, response.Name);
+            Assert.NotNull(response.Id);
+            Assert.NotNull(response.KeyOptions);
+            Assert.NotNull(response.Revision);
+            Assert.NotNull(response.StatusString);
+        }
+
+        [Fact]
+        public async Task GetCollectionRevisionAsync_ShouldThrow_WhenCollectionNotFound()
+        {
+            var exception = await Assert.ThrowsAsync<ApiErrorException>(async () =>
+            {
+                await _collectionApi.GetCollectionRevisionAsync("bogusCollection");
+            });
+            Assert.Equal(HttpStatusCode.NotFound, exception.ApiError.Code);
```

Comment:
Should we also check `ErrorEnum`? It may be be we've done it in other tests so I figured it would be consistent.